core/lib/makeswift/components/site-header/site-header.makeswift.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Checkbox,
+  Image,
+  Link,
+  List,
+  Number,
+  Select,
+  Shape,
+  Slot,
+  TextInput,
+} from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { MakeswiftHeader } from './site-header.client';
+
+export const COMPONENT_TYPE = 'catalyst-makeswift-header';
+
+runtime.registerComponent(MakeswiftHeader, {
+  type: COMPONENT_TYPE,
+  label: 'MakeswiftHeader (private)',
```

Comment:
Should we be using `(private)` in the label? This will show up in the elements panel and this feels like exposing an implementation detail that we shouldn't be exposing